packages/material-components-web/package.json

```diff
@@ -8,6 +8,9 @@
     "material design"
   ],
   "main": "dist/material-components-web.js",
+  "module": "index.js",
+  "sideEffects": false,
+  "types": "dist/mdc.materialComponentsWeb.d.ts",
```

Comment:
it seems odd that this filename is inconsistent with `main`?

Reply:
I think the `.d.ts` filenames are generated automatically by `toCamelCase()` in `scripts/cp-pkgs.js`.

But I agree it feels a little odd; maybe we should add an exception for this case to keep the filenames consistent.

Reply:
Ya, it is a bit odd that we don't follow the pattern of the rest of the components, but it should match the `main` property. I'll make it an exception.